mlir/docs/DataLayout.md

```diff
@@ -289,7 +289,7 @@ The default data layout assumes 8-bit bytes.
 
 ### DLTI Dialect
 
-The [DLTI](Dialects/DLTI.md) dialect provides the attributes implementing
+The [DLTI](https://mlir.llvm.org/docs/Dialects/DLTIDialect/) dialect provides the attributes implementing
```

Comment:
You don't need the full URL, you can use relative links: `../Dialects/DLTIDialect/` or absolute: `/docs/Dialects/DLTIDialect/`.

Reply:
done.